distribution/docs/src/main/jdocs/content/_plugins/tags-filter-plugin-contents.adoc

```diff
@@ -0,0 +1,23 @@
+:type: plugin
+:status: published
+:title: Tags Filter Plugin
+:link: _tags_filter_plugin
+:plugintypes: prefederatedquery
+:summary: Updates queries without filters.
+
+The Tags Filter Plugin updates queries without filters for tags, and adds a default tag of `resource`.
+For backwards compatibility, a filter will also be added to include metacards without any tags attribute.
+
+===== Related Components to Tags Filter Plugin
```

Comment:
My hunch is, once we review all the plugins and docs, we will find that this section will never (or very rarely) be empty. 
Additionally, adding a section on all of the catalog extensions (https://codice.atlassian.net/browse/DDF-3130) will most likely mean all of these plugins will have related components.